packages/zoe/src/contractSupport/priceAuthority.js

```diff
@@ -138,6 +141,64 @@ export function makeOnewayPriceAuthorityKit(opts) {
       return triggerPK.promise;
     };
 
+  const makeMutableQuote = compareAmountsFn =>
+    async function mutableQuoteWhenOutTrigger(amountInArg, amountOutLimitArg) {
+      let amountIn = amountMath.coerce(amountInArg, actualBrandIn);
+      let amountOutLimit = amountMath.coerce(amountOutLimitArg, actualBrandOut);
+
+      /** @type {PromiseRecord<PriceQuote>} */
+      const triggerPK = makePromiseKit();
+
+      const mutableQuote = Far('MutableQuote', {
+        cancel: e => triggerPK.reject(e),
+        updateLevel: (newAmountIn, newAmountOutLimit) => {
+          amountIn = amountMath.coerce(newAmountIn, actualBrandIn);
```

Comment:
Can you handle when `newAmountIn` succeeds in coercing but `newAmountOutLimit` fails the coercion?

Reply:
done